wremnants/datasets/datagroups.py

```diff
@@ -585,29 +597,29 @@ def sum_and_unc(h):
             
         return df
 
-    def set_rebin_action(self, axes, ax_lim=[], ax_rebin=[], ax_absval=[]):
+    def set_rebin_action(self, axes, ax_lim=[], ax_rebin=[], ax_absval=[], rebin_before_selection=False, epsilon=10e-10):
         if len(ax_lim) % 2 or len(ax_lim)/2 > len(axes) or len(ax_rebin) > len(axes):
             raise ValueError("Inconsistent rebin or axlim arguments. axlim must be at most two entries per axis, and rebin at most one")
-
+        self.rebinBeforeSelection = rebin_before_selection
```

Comment:
Should this be the default, or not?

Reply:
I think by default the rebinning should happen after the selection (fakerate estimation) because I think the intuitive expectation is that the rebinning does not change the yields and is only for visualization.